qemu/tests/cfg/migrate.cfg

```diff
@@ -11,7 +11,8 @@
     mig_timeout = 3600
     ping_pong = 1
     pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
-    cmds_installed_host = "ksmtuned"
+    !Host_RHEL.m6,Host_RHEL.m7,Host_RHEL.m8:
+        package_yum_install = "ksmtuned"
```

Comment:
What do you think about change the param name a bit,   especially it is for RHEL9+ but named `package_yum_install`

Reply:
> What do you think about change the param name a bit, especially it is for RHEL9+ but named `package_yum_install`
Changed this param name to "depends_pkgs".